### DISCUSSION TITLE
How to get the Video duration from UploadFile object in request?

### DISCUSSION THREAD

**Discussion by adityashanbhog** [fastapi/fastapi#7310] [Questions] [answered]
The use is to limit the user from uploading video having a duration greater than 1 minute.

**Comment by tiangolo** [comment, marked as answer]
I imagine you solved your problem, so thanks for coming back to close the issue!

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
